source/llassetgen-rendering/main.cpp

```diff
@@ -0,0 +1,383 @@
+#include <algorithm>
+#include <iostream>
+#include <string>
+
+#include <ft2build.h>  // NOLINT include order required by freetype
+#include FT_FREETYPE_H
+
+#include <QApplication>
+#include <QBoxLayout>
+#include <QFormLayout>
+#include <QGroupBox>
+#include <QLabel>
+#include <QLineEdit>
+#include <QMainWindow>
+#include <QResizeEvent>
+#include <QValidator>
+
+#include <glm/glm.hpp>
+
+#include <glbinding/ContextInfo.h>
+#include <glbinding/Version.h>
+#include <glbinding/gl/gl.h>
+
+#include <globjects/base/File.h>
+#include <globjects/globjects.h>
+#include <globjects/logging.h>
+
+#include <globjects/Buffer.h>
+#include <globjects/Program.h>
+#include <globjects/Shader.h>
+#include <globjects/Texture.h>
+#include <globjects/VertexArray.h>
+#include <globjects/VertexAttributeBinding.h>
+
+#include <llassetgen/llassetgen.h>
+
+#include "WindowQt.h"
+
+using namespace gl;
```

Comment:
I don't know how many types in this file are from the `gl` namespace, but if its not too many a few `using gl::typename` lines would do the same and not pull in all members of the namespace.

Reply:
There are currently 20 identifiers that need that namespace, and they all start with gl. So instead of `GL_TEXTURE_2D` I'd need to write `gl::GL_TEXTURE_2D`; or `glClear becomes `gl::glClear`.
So I'd rather use this namespace as it is. 

Well, I am not sure... are 20 identifiers few or many?

Reply:
Oh, ok. If that namespace just contains the usual opengl symbols (which are prefixed with `GL_`, `gl`, etc. anyway), then the `using namespace` is fine.

Reply:
Yes, we designed the `gl*` namespaces to include only the official symbols from the OpenGL API of their respective version (namespace `gl` includes all).